src/CMakeLists.txt

```diff
@@ -23,9 +23,9 @@ SET(CMAKE_CXX_FLAGS_COMMON
      -Wextra \
      -Werror \
      -pedantic \
-     -g \
-     -fsanitize=address \
-     -fsanitize=undefined")
+     -g")
```

Comment:
Oh sorry I mistakenly included this. This is just to let my compiler not complain about the sanitizer issue.